libs/live-dmk/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "@ledgerhq/live-dmk",
+  "version": "0.1.0",
+  "description": "Ledger Device Managment Kit transport module",
```

Comment:
I didn't see this but let's change it later, the description is not really accurate.
Also I think we can put `"private": true`